models/domains_tlds.php

```diff
@@ -218,9 +218,22 @@ public function add(array $vars)
      */
     private function createPackage(array $vars)
     {
-        Loader::loadModels($this, ['Currencies', 'Languages', 'Companies']);
+        Loader::loadModels($this, ['ModuleManager', 'Currencies', 'Languages', 'Companies']);
         Loader::loadHelpers($this, ['Form']);
 
+        // Validate if the TLD is supported by the provided module
+        if (isset($vars['module_id'])) {
```

Comment:
This can be an input validation rule using an anonymous function like the ['module_id']['service'] rule does.